backend/open_webui/main.py

```
@@ -318,6 +319,11 @@ async def get_response(self, path: str, scope):
             return await super().get_response(path, scope)
         except (HTTPException, StarletteHTTPException) as ex:
             if ex.status_code == 404:
+                # Don't serve index.html for API routes
+                if path.startswith(
+                    ("api/", "mcp/", "ollama/", "openai/", "ws/", "health", "oauth/")
+                ):
+                    raise ex
```

Ruff (0.12.2)

Use `raise` without specifying exception name

Remove exception name

(TRY201)